gpt_bpe.go

```diff
@@ -102,6 +104,7 @@ const VOCAB_ID_PILE = "pile-tokenizer"
 const VOCAB_ID_CLIP = "clip-tokenizer"
 const VOCAB_ID_NERDSTASH_V1 = "nerdstash_v1-tokenizer"
 const VOCAB_ID_NERDSTASH_V2 = "nerdstash_v2-tokenizer"
+const VOCAB_ID_LLAMA = "llama-tokenizer"
```

Comment:
Is LLama-2 the same as Llama?

Reply:
Practically, yes. Actually, no. It's a bit of a judgement call, but I think we should separate them in a followup PR.